lib/gnarails/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Gnarails
-  VERSION = "3.0.0"
+  VERSION = "3.1.0"
 end
```

Comment:
Ideally, I think we'd have this change as the last commit on the branch named 3.1.0, and use that as the base branch here. That would let us add any other upgrades we want to include to the release, which seems helpful. We don't really follow any kind of strong pattern for this repo, so there's room to set up some good practices as we make improvements. I also REALLY liked your labelled branch name.

Reply:
I'm only mentioning this because I don't think we need to rush out a minor version for this repo. There are probably some other small things we could add here, and I'd like to coordinate that with @efertsch since he's managing the product sprint.

Reply:
Yeah, no worries.  Thanks for the feedback.  I was expecting feedback on the deployment and versioning processes.

Reply:
@nialbima great call. I think that we should definitely leave versioning to a separate piece of work. I am going to be adding the ticket for the ruby upgrade today.